src/screens/Endorsements/Endorsements.js

```diff
@@ -46,7 +22,7 @@ const Endorsements = () => {
             </Grid>
           ))}
         </EndorsersWrapper>
-        <ShareBlock />
+        {social && <ShareBlock />}
```

Comment:
is this ever true?

Reply:
Ah, should be when standalone. Updating!